query_csv.py

```diff
@@ -0,0 +1,369 @@
+import csv
+import copy
+from itertools import islice
+from datetime import datetime as dt
+
+
+current_month = dt.now().strftime('%B')
+
+
+def read_country_data(month=current_month):
```

Comment:
This is a really long function. Try to split it up and give them descriptive name so it is clear what they are doing.

Reply:
Done